source/thisTarget.ts

```diff
@@ -61,10 +61,6 @@ export function getTabDataStatus(): typeof tabDataStatus {
 }
 
 const storeTabData = once(async () => {
-  if (tabDataStatus !== "needed") {
-    return;
-  }
```

Comment:
I believe this is safe to remove given the `once` wrapping around this function, but there might be cases where we are now making more calls through to to the `messenger("__getTabData")` method that we weren't already.

I'll double check and see if this has any impact on additional errors / calls.